docker/local-metadata/docker-compose.yml

```diff
@@ -125,9 +126,11 @@ services:
     expose:
       - 8585
       - 8586
+      - 5005
```

Comment:
Can we add this for all the docker-compose.yml files ?
development -
- docker/local-metadata/docker-compose-postgres.yml

production -
- docker/metadata/docker-compose.yml
- docker/metadata/docker-compose-postgres.yml

Reply:
No